utils/config/i18n.ts

```diff
@@ -25,42 +25,16 @@ export const langsFlags = [
   },
 ]
 
-// const md = MarkdownIt({
-//   breaks: false,
-// })
+const md = MarkdownIt({
+  breaks: false,
+})
 
-function loadLocaleMessages() {
-  // File containing data common to ALL languages
-  // const allLangDataFile = 'all_lang.json'
-  // const locales = require.context(
-  //   '../../locales',
-  //   true,
-  //   /[A-Za-z0-9-_,\s]+\.json$/i
-  // )
-  // const messages = {}
-  // locales.keys().forEach((key) => {
-  //   if (key === allLangDataFile) {
-  //     return
-  //   }
-  //   const matched = key.match(/([A-Za-z0-9-_]+)\./i)
-  //   if (matched && matched.length > 1) {
-  //     const locale = matched[1]
-  //     messages[locale] = locales(key)
-  //   }
-  // })
-  // return messages
-  return {}
-}
-
-export default () => {
-  return {
-    locale: process.env.VUE_APP_I18N_LOCALE || 'en',
-    fallbackLocale: process.env.VUE_APP_I18N_FALLBACK_LOCALE || 'en',
-    silentTranslationWarn: true,
-    modifiers: {
-      // md: (str) => md.renderInline(str),
-      common: (str) => str.split('.').reduce((o, i) => o[i], commonData),
-    },
-    messages: loadLocaleMessages(),
```

Comment:
What about this? it's needed to parse some FAQ content

Reply:
Ok made this work on my new pr. Replaced webpacks require.context with import.glob from vite

Reply:
#7045